tools/cockpit.spec

```diff
@@ -513,7 +513,7 @@ Requires: expect
 Requires: libvirt
 Requires: libvirt-client
 Requires: libvirt-daemon
-%if 0%{?rhel}%{?centos} == 0 || 0%{?rhel} >= 8 || 0%{?centos} >= 8
+%if 0%{?rhel} >= 8 || 0%{?centos} >= 8 || 0%{fedora} >= 27
```

Comment:
the fedora clause is missing the ?

Reply:
Yes it was. It worked anyway for me because the `?` only affects the output when the macro cannot be found.

Thanks for the catch! Updated.